blueprints/reducer/files/__root__/reducers/__name__.js

```diff
@@ -0,0 +1,38 @@
+// __name__ REDUCER
```

Comment:
arguably we could remove the APP / ADDON abstraction because ember-cli now supports the `--dummy` flag but the `__name__.js` is still the _right way_ to name generated files.